packages/app-shell/src/components/Breadcrumbs/TrackBreadcrumb.tsx

```diff
@@ -0,0 +1,21 @@
+import { useContext, useEffect } from 'react';
+import { Props } from '@airbnb/lunar/lib/components/Breadcrumbs/Breadcrumb';
+import AppContext from '../AppContext';
+
+export default function TrackBreadcrumb({ label, ...props }: Props) {
+  const context = useContext(AppContext);
+
+  if (!context) {
+    return null;
+  }
+
+  useEffect(() => {
+    const id = context.addBreadcrumb(label, props);
+
+    return () => {
+      context.removeBreadcrumb(id);
+    };
+  }, [label]);
```

Comment:
do you also need to pass in `props` (based on that other lint error i saw yesterday)?